mw-webapp/src/logic/aboutProjectPage/AboutProjectPage.module.scss

```diff
@@ -1,18 +1,90 @@
 @import "src/styles/variables";
 
 .pageWrapper {
-  display: flex;
-  flex-direction: column;
   align-items: center;
   padding: 0 $gapMedium;
   margin-top: $marginMedium;
+  gap: $gapBig;
+}
+
+.title {
+  color: var(--secondaryTextButtonColor);
+  text-align: center;
+}
+
+.subtitle {
+  color: var(--primaryTextColor);
+  text-align: center;
+  margin-bottom: $marginSmall;
+}
+
+.aboutBlock {
+  max-width: $maxWidth;
+  align-items: center;
+  gap: $gapSmall;
+  color: var(--secondaryTextColor);
+  background-color: var(primarysBackgroundColor);
+}
+
+.aboutDescription {
+  display: flex;
+  align-items: center;
+  margin-top: $marginBig;
+  background-color: var(--primaryBackgroundColor);
 }
 
-.projectDescription {
-  max-width: 600px;
+.aboutTextBlock {
+  flex: 1 1 70%;
+  padding: $paddingMedium;
+  gap: $gapBig;
+  color: var(--primaryTextColor);
+  background-color: var(--primaryBackgroundColor);
 }
 
+.logoAbout {
+  flex: 1 1 30%;
+  padding: $paddingMedium;
+  background-color: var(--primaryBackgroundColor);
+  @media (max-width: 850px) {
+    display: none;
+  }
+}
+
+.ourTeamBlock {
+  color: var(--secondaryTextColor);
+  background-color: var(--tertiaryBackgroundColor);
+  width: 100%;
+  padding: $paddingBig;
+  gap: $gapLarge;
+  text-align: center;
+}
+
+.ourTeamMembers {
+  display: flex;
+  justify-content: center;
+  gap: $gapLarge;
+  overflow-x: auto;
+  &::-webkit-scrollbar {
+    height: 8px;
+    background-color: var(--tertiaryBackgroundColor);
+    border-radius: 10px
+  }
+  &::-webkit-scrollbar-thumb {
+    background-color: var(--secondaryTextColor);
+    height: 8px;
```

Comment:
could you tell more about it? share a link please

Reply:
done